bootstrap/templates/bootstrap/field.html

```diff
@@ -2,8 +2,10 @@
     {{ field }}
 {% else %}
     <div id="{{ field.auto_id }}-group" class="form-group field-{{ field_class }} widget-{{ widget_class }}{% if field.errors %} has-error{% endif %}{% if field.field.required %} required{% endif %}{% if extra_classes %} {{ extra_classes }}{% endif %}">
-        {% if show_label and not is_checkbox %}
+        {% if show_label and not is_checkbox and not is_radioselect%}
         <label for="{{ field.auto_id }}">{{ field.label }}</label>
```

Comment:
We may as well add `id="{{ field.auto_id }}-label"` to this regardless of the widget type, so that other types could use it as well.